src/util/math.hpp

```diff
@@ -9,6 +9,17 @@ namespace big::math
 		return (float)radian;
 	}
 
+	inline int meters_to_feet(int meters)
```

Comment:
Doesn't `mps_to_speed` or `speed_to_mps` already do this?
Even if it doesn't i would rather add it there instead of adding it here.
Also why use an `int` instead of a `float`?